src/storage/page_buffer.c

```diff
@@ -1429,6 +1474,11 @@ pgbuf_initialize (void)
       goto error;
     }
 
+#if defined(SERVER_MODE)
+  pthread_mutex_init (&pgbuf_Pool.show_status_mutex, NULL);
+#endif
+  pgbuf_Pool.show_status.old.print_out_time = time (NULL);
+
```

Comment:
I think, you need to initialize variables of show_status.old as the following.

pgbuf_Pool.show_status.old.num_hit = 0;
  pgbuf_Pool.show_status.old.num_page_request = 0;
  pgbuf_Pool.show_status.old.num_pages_created = 0;
  pgbuf_Pool.show_status.old.num_pages_written = 0;
  pgbuf_Pool.show_status.old.num_pages_read = 0;

Reply:
@mhoh3963 Since the global variable pgbuf_Pool is set to 0 in the pgbuf_initialize() function, a separate initialization is not necessary.

Reply:
OK, i got it.